statsmodels/regression/_prediction.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Dec 19 11:29:18 2014
+
+Author: Josef Perktold
+License: BSD-3
+
+"""
+
+import numpy as np
+from scipy import stats
+
+# this is similar to ContrastResults after t_test, partially copied and adjusted
+class PredictionResults(object):
+
+    def __init__(self, predicted_mean, var_pred_mean, var_resid,
+                 df=None, dist=None, row_labels=None):
+        self.predicted_mean = predicted_mean
+        self.var_pred_mean = var_pred_mean
+        self.df = df
+        self.var_resid = var_resid
+        self.row_labels = row_labels
+
+        if dist is None or dist == 'norm':
+            self.dist = stats.norm
+            self.dist_args = ()
+        elif dist == 't':
+            self.dist = stats.t
+            self.dist_args = (self.df,)
+        else:
+            self.dist = dist
+            self.dist_args = ()
+
+    @property
+    def se_obs(self):
+        return np.sqrt(self.var_pred_mean + self.var_resid)
+
+    @property
+    def se_mean(self):
```

Comment:
I think our convention is (going to be) `mean_se`   we have `effect_se` somewhere

similar obs_se,  `obs` is not really good, but I don't know an alternative